data/convert_to_triples.R

```diff
@@ -0,0 +1,442 @@
+      description = paste0("\"", description, "\"", "@en"),
+      Parent = paste0("<", Parent, ">"),
+      SubClassOf = Parent
+      exactMatch = ifelse(!is.na(exactMatch), paste0("<", exactMatch, ">"), NA)
+    ) %>%
+    rename(
+      Subject = Entity,
+      `<http://www.w3.org/2000/01/rdf-schema#label>`= label,
+      `<http://purl.org/dc/terms/description>` = description,
+      `<http://www.w3.org/2004/02/skos/core#broader>` = Parent,
+      `<http://www.w3.org/2000/01/rdf-schema#subClassOf>` = SubClassOf,
+      `<http://www.w3.org/2004/02/skos/core#exactMatch>` = exactMatch
+    ) %>%
+    pivot_longer(cols = c(2:5)) %>% 
+    rename(
+      Predicate = name,
+      Object = value
+    ) %>% 
+  filter(!is.na(Object))
+
+reformatted_ontology <- read_csv("data/ontology_links.csv") %>%
+  select(Entity, label, description, identifier, inScheme, prefix) %>%
+  mutate(across(where(is.character), \(x) stringr::str_replace_all(x, "\"", "'"))) %>%
+  mutate(
+    Entity = paste0("<", Entity, ">"),
+    label = paste0("\"", label, "\"", "^^<xsd:string>"),
+    description = ifelse(!is.na(description), paste0("\"", description, "\"", "@en"), NA),
+    identifier = str_replace(identifier, "^[:alpha:]+\\:", ""),
+    identifier = paste0("\"", identifier, "\"", "^^<xsd:string>"),
+    inScheme = paste0("\"", inScheme, "\"", "^^<xsd:string>")
+  ) %>%
+  select(-prefix) %>%
+  rename(
+    Subject = Entity,
+    `<http://www.w3.org/2000/01/rdf-schema#label>`= label,
+    `<http://purl.org/dc/terms/description>` = description,
+    `<http://purl.org/dc/elements/1.1/identifier>` = identifier,
+    `<http://www.w3.org/2004/02/skos/core#inScheme>` = inScheme
+  ) %>%
+  pivot_longer(cols = c(2:5)) %>% 
+  rename(
+    Predicate = name,
+    Object = value
+  ) %>% 
+  filter(!is.na(Object))
+  
+ontology_links <- read_csv("data/ontology_links.csv")
+reviewers <- read_csv("data/APD_reviewers.csv")
+references <- read_csv("data/APD_references.csv")
+units_csv <- read_csv("data/APD_units.csv")
+hierarchy <- read_csv("data/APD_trait_hierarchy.csv")
+
+reformatted_traits <- read_csv("data/APD_traits.csv") %>% 
+  mutate(across(where(is.character), \(x) stringr::str_replace_all(x, "\"", "'"))) %>%
+  mutate(
+    Entity =  paste0("<https://github.com/traitecoevo/", identifier, ">"),
+    trait = paste0("\"", trait, "\"", "^^<xsd:string>"),
+    label = paste0("\"", label, "\"", "^^<xsd:string>"),
+    preflabel = label,
+    description_encoded = ifelse(!is.na(description_encoded), paste0("\"", description_encoded, "\"", "@en"), NA),
+    description = ifelse(!is.na(description), paste0("\"", description, "\"", "@en"), NA),
+    comments = ifelse(!is.na(comments), paste0("\"", comments, "\"", "@en"), NA),
+    inScheme = paste0("\"", inScheme, "\"", "^^<xsd:string>"),
+    type = paste0("<", ontology_links$Entity[match(type, ontology_links$identifier)], ">"),
+    min = ifelse(!is.na(min), paste0("\"", min, "\"", "^^<xsd:double>"), NA),
```

Comment:
numeric values do need types, and these must be full URIs `<http://www.w3.org/2001/XMLSchema#double>`